src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -417,40 +427,43 @@ const MeasureGroups = () => {
                   {...formikFieldProps}
                   helperText={error}
                   error={!!error && !!touched}
+                  canEdit={canEdit}
                 />
               </Fragment>
             );
           })}
           <br />
         </Content>
       </Grid>
-      <GroupFooter>
-        <GroupActions />
-        <PopulationActions>
-          <ButtonSpacer>
-            <Button
-              style={{ background: "#424B5A" }}
-              type="submit"
-              buttonTitle="Save"
-              data-testid="group-form-submit-btn"
-              disabled={!(formik.isValid && formik.dirty)}
-            />
-          </ButtonSpacer>
-          <ButtonSpacer>
-            <Button type="button" buttonTitle="Cancel" variant="white" />
-          </ButtonSpacer>
-          <ButtonSpacer>
-            <span
-              tw="text-sm text-gray-600"
-              data-testid="save-measure-group-validation-message"
-            >
-              {MeasureGroupSchemaValidator.isValidSync(formik.values)
-                ? ""
-                : "You must set all required Populations."}
-            </span>
-          </ButtonSpacer>
-        </PopulationActions>
-      </GroupFooter>
+      {canEdit && (
+        <GroupFooter>
+          <GroupActions />
+          <PopulationActions>
+            <ButtonSpacer>
+              <Button
+                style={{ background: "#424B5A" }}
+                type="submit"
+                buttonTitle="Save"
+                data-testid="group-form-submit-btn"
+                disabled={!(formik.isValid && formik.dirty)}
+              />
+            </ButtonSpacer>
+            <ButtonSpacer>
+              <Button type="button" buttonTitle="Cancel" variant="white" />
```

Comment:
I know the cancel button doesn't do anything now, but not sure if we want to hide this as on most pages cancel is how we navigate backwards.

Reply:
@nmorasb For this one, it seems to be under Measure Groups tab, along with Details, CQL Editor and Test Cases tab, so user can still go to different pages. I'll ask about this at today's meeting. Thanks a lot for the feedback!

Reply:
@nmorasb I asked Kim and she suggested hide all Cancel buttons, so I made more changes for hiding the Cancel buttons for measure non-owners. Thanks Nate for pointing out the inconsistency.